src/mbgl/tile/geometry_tile_data.cpp

```diff
@@ -112,8 +112,6 @@ static Feature::geometry_type convertGeometry(const GeometryTileFeature& geometr
         );
     };
 
-    GeometryCollection geometries = geometryTileFeature.getGeometries();
```

Comment:
@julianrex @pozdnyakov was there a need to pass `const GeometryCollection& geometries` all the way here when we already have constref to geometryTileFeature?

I think the problem was on line 115 where we made copy. Should have been:

`const GeometryCollection& geometries = geometryTileFeature.getGeometries();`